packages/grid/_modules_/grid/models/colDef/gridNumericOperators.ts

```diff
@@ -134,3 +134,8 @@ export const getGridNumericColumnOperators = (): GridFilterOperator[] => [
     },
   },
 ];
+
+/**
+ * @deprecated Use `getGridNumericOperators` instead.
+ */
+export const getGridNumericColumnOperators = getGridNumericOperators;
```

Comment:
Renamed to `getGridNumericOperators` to be consistent with the other operator functions.

Reply:
You could put a deprecated message
```
export const getGridNumericColumnOperators = () => {
	// Warning message here
	return getGridNumericOperators()
}
```